### PR TITLE
Merge back 1.4.7 release

### PR DESCRIPTION
**Pull request by matYang**
## Solution

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-2.10.0-ccip1.4.5-release
```
Comment (RensR) [inline VERSION:1]: This feels wrong

Comment (matYang) [reply VERSION:1]: you are fast! just found it too and updated

File: VERSION [hunk 2]
```diff
@@ -1 +1 @@
...
+2.10.0-ccip1.4.7-release
```
Comment (mateusz-sekara) [inline VERSION]:
```suggestion
2.10.0-ccip1.4.7-beta.0
```

BIX sometimes deploys from ccip-develop directly, having `beta.0` is safer

**Review by mateusz-sekara — APPROVED** [review]

File: VERSION (hunk 2 above)
Comment (matYang) [reply VERSION]: bumped to 2.10.0-ccip1.4.9-beta.0
this branch will bring in the the 1.4.7 changes into ccip-develop
1.4.8 changes are already in ccip-develop (was cherry-picked into release branch)

**Pull request merged**